COPY/usr/bin/manageiq-messaging-ready

```diff
@@ -12,7 +12,7 @@ end
 
 def messaging_ready?(msg_yaml, env = "production")
   host, port = msg_yaml[env].values_at("hostname", "port")
-  host ||= "localhost"
+  host ||= "127.0.0.1"
```

Comment:
If we need to replace `localhost` with `127.0.0.1` I think it is better to do it before we write the messaging.yml so we don't have to do this everywhere else

Reply:
Is this because ssl does verification of the hostname on the certificate?  I'm curious how this works for an ipv4 address.

Reply:
@agrare yes that is done here https://github.com/ManageIQ/manageiq-appliance_console/pull/198, this is for the default host value in the case that `hostname` is empty in the messaging.yml hence `||=`

Reply:
https://github.com/ManageIQ/manageiq-appliance/pull/368#discussion_r975706602

Reply:
D'oh need to not multitask-review

@jrafanie when you specify an IP address we disable cert hostname checking